db/migrate/20171030205901_add_list_id_to_tasks.rb

```diff
@@ -0,0 +1,5 @@
+class AddListIdToTasks < ActiveRecord::Migration[5.1]
+  def change
+    add_column :tasks, :list_id, :integer
```

Comment:
One more, can you have tasks that don't belong to a list?

Reply:
For the purposes of this app, there shouldn't be tasks without a list.